video.c

```diff
@@ -2910,27 +2910,28 @@ static const bitmap_layer_render_struct bitmap_mode_renderers[3] =
     }                                                                         \
   }                                                                           \
   else                                                                        \
-                                                                              \
-  if((obj_x + obj_width) >= end)                                              \
-  {                                                                           \
-    pixel_run = end - obj_x;                                                  \
-    if((s32)pixel_run > 0)                                                    \
+  {                                                                           \                                                                            \
```

Comment:
build error: remove extra "\" in this line

Reply:
@negativeExponent you are correct, this should be removed before applying the commit